### ISSUE TITLE
Build fails: use of undeclared identifier 'environ'

### ISSUE THREAD

**Issue by yurivict** [psi4/psi4#851]
Version 1.1 fails:

```
/usr/ports/science/psi4/work/psi4-1.1/psi4/src/psi4/libparallel/process.cc:69:9: error: use of undeclared identifier 'environ'
    if (environ) {
        ^
/usr/ports/science/psi4/work/psi4-1.1/psi4/src/psi4/libparallel/process.cc:70:16: error: use of undeclared identifier 'environ'
        while (environ[i] != NULL) {
               ^
/usr/ports/science/psi4/work/psi4-1.1/psi4/src/psi4/libparallel/process.cc:71:52: error: use of undeclared identifier 'environ'
            std::vector <std::string> strs = split(environ[i], "=");
                                                   ^
```

Found on the FreeBSD 11.1.
'environ' needs to be declared.

**Comment by loriab** [comment]
Looks the OS is supposed to provide it, [except on Apple](https://github.com/psi4/psi4/blob/master/psi4/src/psi4/libpsi4util/process.cc#L49-L53). Maybe that ifdef needed to be more specific to BSD than Apple.

Though that code is still present in master, accessing the environment is not, so it should be removed. For now compiling 1.1, please just adjust that ifdef or whatever is necessary.

Thanks for the report.

**Comment by yurivict** [comment]
> please just adjust that ifdef or whatever is necessary.

That's what I did.

Thanks!

**Comment by jturney** [comment]
This was fixed in PR #856
